manifests/m/Microsoft/NuGet/6.4.0/Microsoft.NuGet.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.NuGet
+PackageVersion: 6.4.0
+MinimumOSVersion: 10.0.0.0
+Commands:
+- nuget
+Installers:
+- Architecture: x86
+  InstallerType: portable
+  InstallerUrl: https://dist.nuget.org/win-x86-commandline/v6.4.0/nuget.exe
+  InstallerSha256: 26730829B240581A3E6A4E276B9ACE088930032DF0C680D5591BECCF6452374E
+  AppsAndFeaturesEntries:
+  - DisplayName: NuGet CLI
+    DisplayVersion: 6.3.1.1

```

```suggestion
    DisplayVersion: 6.4.0.123
```
Invoking the NuGet command shows its version
```pwsh
 ~/Downloads ./nuget
NuGet Version: 6.4.0.123
usage: NuGet <command> [args] [options]
Type 'NuGet help <command>' for help on a specific command.
```